_posts/2021-04-18-api-conversion.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title:  "Path to CRD v1"
+date: 2021-04-18
+author: jakub_stejskal
+---
+In Strimzi 0.22 we introduced the `v1beta2` API version to prepare Strimzi for Kubernetes 1.22. The `v1beta2` API version replaces the `v1beta1` and `v1alpha1` API versions.
+This post shows you how to perform an API conversion to `v1beta2` to be ready for the next Strimzi releases.
+
+<!--more-->
+  
+### New API versions
+
+By introducing API version `v1beta2` for Strimzi resources, current Strimzi users must migrate from the old `v1beta1`  and `v1beta2` versions.
```

Comment:
> the old `v1beta1` and `v1beta2` versions
this makes an implication that `v1beta2` is an old version. I think you should rephrase this sentence.

Reply:
ye, Paolo is right. here should be `v1alpha1` and `v1beta1`